ios/Podfile.lock

```diff
@@ -1,27 +0,0 @@
-PODS:
-  - Capacitor (1.1.1):
-    - CapacitorCordova (= 1.1.1)
-  - CapacitorCordova (1.1.1)
-  - RadarSDK (3.0.1)
```

Comment:
should we be bumping this to 3.3.1 as well?

Reply:
This file was deleted.